lib/options-manager.js

```diff
@@ -52,6 +52,9 @@ resolveFrom.silent = (moduleId, fromDirectory) => {
 	} catch { }
 };
 
+// TODO: Use `resolveModule(normalizePackageName(name), import.meta.url);` when moving to ESM then to `import.meta.resolve(normalizePackageName(name))` when supported
+const resolveLocalConfig = name => resolveModule(normalizePackageName(name, 'eslint-config'), require.main.filename);
```

Comment:
> doesn't resolveModule(normalizePackageName(name), import.meta.url); also need to search from the parent directory of import.meta.url?

[`resolveModule`](https://github.com/eslint/eslintrc/blob/9fb9c5da9b9540123a93b096557e9c249a2671ef/lib/shared/relative-module-resolver.js#L26-L43) just wraps `require("module").createRequire(path).resolve(name)` which is the same code that is recommended to be used in a related [StackOverflow question](https://stackoverflow.com/a/62499498/8384910).

> Wouldn't it be more correct to use path.resolve('..', __dirname) instead of require.main.filename?

Better still, we could just use `name => require.resolve(normalizePackageName(name))`.